Bryant_Trinh/Function Basic 2.py

```diff
@@ -0,0 +1,39 @@
+#1
+def function(cd):
+	arr=[]
+	for i in range(cd, 0):
```

Comment:
range(cd,0,-1) //Moving down
Append values to the arr //not pop from an empty array